### PR TITLE
Some Javascript related links added

### PR DESCRIPTION
**Pull request by me-shaon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,12 +73,18 @@ Because we think it might help others as well we open-sourced it. If you know so
 
 * [MDN - Mozilla Developer Network](https://developer.mozilla.org/de/)
 * [Talks To Help You Become A Better Front-End Engineer](https://www.smashingmagazine.com/2012/12/talks-to-help-you-become-a-better-front-end-engineer-in-2013/)
-* [Eloquent Javascript](http://eloquentjavascript.net/)
 * [Netflix UI Engineers Youtube Channel](https://www.youtube.com/channel/UCGGRRqAjPm6sL3-WGBDnKJA)
 * [Twitter Bootstrap](http://getbootstrap.com)
-* [Babel.js - ES6 today](https://babeljs.io/)
 * [Don't make me think - Web Usability](https://www.amazon.de/Dont-make-think-Usability-intuitive/dp/3826697057/ref=sr_1_1?ie=UTF8&qid=1476888649&sr=8-1&keywords=dont+make+me+think)
 
+### JavaScript
```
Comment (daniel-mueller) [inline README.md]: Hi,
we use two hashes for minor headlines

**Comment by me-shaon** [comment]
Hi, thanks for the reply. I've updated the tag.

File: README.md [hunk 2]
```diff
@@ -73,12 +73,18 @@ Because we think it might help others as well we open-sourced it. If you know so
...
 * [MDN - Mozilla Developer Network](https://developer.mozilla.org/de/)
 * [Talks To Help You Become A Better Front-End Engineer](https://www.smashingmagazine.com/2012/12/talks-to-help-you-become-a-better-front-end-engineer-in-2013/)
-* [Eloquent Javascript](http://eloquentjavascript.net/)
 * [Netflix UI Engineers Youtube Channel](https://www.youtube.com/channel/UCGGRRqAjPm6sL3-WGBDnKJA)
 * [Twitter Bootstrap](http://getbootstrap.com)
-* [Babel.js - ES6 today](https://babeljs.io/)
 * [Don't make me think - Web Usability](https://www.amazon.de/Dont-make-think-Usability-intuitive/dp/3826697057/ref=sr_1_1?ie=UTF8&qid=1476888649&sr=8-1&keywords=dont+make+me+think)
 
+## JavaScript
+* [Learn JavaScript Essentials](https://medium.com/javascript-scene/learn-javascript-b631a4af11f2#.ggqhkheuz)
```
Comment (daniel-mueller) [inline README.md]: Please add a newline between headline and links

**Comment by me-shaon** [comment]
New line added. Thanks. :+1:

**Pull request merged**